tests/test_sphere.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import pytest
+import unittest
+from ddt import ddt, data, unpack
+
+import numpy as np
+from broni.shapes.primitives import Sphere
+
+
+@ddt
+class TestSphere(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    @data(
+        (np.array([[1, 1, 1]]), Sphere(0, 0, 0, 10),
+         np.array([True])),  # one point
+        (np.array([[0, 0, 0], [1, 1, 1]]), Sphere(0, 0, 0, 0),
+         np.array([True, False])),  # empty sphere, only center dot OK
```

Comment:
should we expect equality of one (floating point) point with a (float) zero-radius sphere? I wouln't think so... to me a zero radius sphere does not make sense

Reply:
Yes, if this is forbidden for Cuboide I see no reason to allow it for a Sphere

Reply:
OK